manifests/candlepin.pp

```diff
@@ -106,19 +98,13 @@
       ensure  => file,
       content => $keystore_password,
       owner   => 'root',
-      group   => $group,
-      mode    => '0440',
+      group   => 'root',
```

Comment:
Is this file only temporary for the Puppet commands?

Reply:
Effectively yes, and for debugging the keystore/truststore.